### PR TITLE
changed header to new theme

### PR DESCRIPTION
**Pull request by jvberdec**
Still needed: implementing new theme into the rest of the site (forms and about us)

### CODE REVIEW

**Comment by jvberdec** [comment]
@Quetourah Let me know how it looks now.

**Comment by Quetourah** [comment]
> @Quetourah Let me know how it looks now. It cleared quite a bit of space in the home file.

Awesome! Looks great 😺 .

**Comment by jvberdec** [comment]
> > @Quetourah Let me know how it looks now. It cleared quite a bit of space in the home file.
> 
> Awesome! Looks great 😺 .

Now we can have a nicer time putting things in the home page! Thanks.

**Comment by Quetourah** [comment]
> > > @Quetourah Let me know how it looks now. It cleared quite a bit of space in the home file.
> > 
> > 
> > Awesome! Looks great 😺 .
> 
> Now we can have a nicer time putting things in the home page! Thanks.

np :)!

File: cards/templates/cards/AboutUs.html [hunk 1]
```diff
@@ -1,32 +1,28 @@
-{% extends "cards/homepage.html" %}
+{% extends "cards/index.html" %}
 {% load crispy_forms_tags %}
```
Comment (aqsa505) [inline cards/templates/cards/AboutUs.html:2]: we don't need "Crispy forms tag" in About Us page

Comment (aqsa505) [inline cards/templates/cards/AboutUs.html:2]: we don't need "crispy forms tag" in About US page

Comment (jvberdec) [reply cards/templates/cards/AboutUs.html:2]: Thanks! I'll remove.

**Pull request merged**